assignments/arrays.js

```diff
@@ -56,41 +56,67 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // Example for loop:
 
-// arr = [1,2,3,4];
-// for (i = 0; i < arr.length; i++) {
-//     arr[i]; // 1,2,3,4
-// }
+arr = [1,2,3,4];
+for (i = 0; i < arr.length; i++) {
+    arr[i]; // 1,2,3,4
+}
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}` );
```

Comment:
This works for now. But what if a few of the cars were deleted from the array, and you didn't know any more at which index the car with id 33 was stored? How could you use a for loop to find this same answer, that will account for any changes made to the array?